tests/syntax/test_errors.py

```diff
@@ -24,49 +25,25 @@ def test_bad_extension(tmpdir):
 
 ### Parse errors during token translation
 
-## Operators used in Python but not Scenic
```

Comment:
those tests were for the token translator and not relevant anymore

Reply:
Agreed, but we should also change the preceding comment (on line 25) since there is no token translation phase any more.